README.md

```diff
@@ -3,20 +3,19 @@
 [![Documentation Status](https://readthedocs.com/projects/canonical-charmed-hpc/badge/?version=latest)](https://canonical-charmed-hpc.readthedocs-hosted.com/en/latest/?badge=latest)
 [![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#hpc:ubuntu.com)
 
-Charmed HPC's documentation 📑🔍
+Charmed HPC's documentation
 
 For more information on how to access or contribute to Charmed HPC's
 documentation, check out the [Getting Started](#-getting-started) and
-[Development](#️-development) sections below 👇
+[Development](#️-development) sections below.
 
-## ✨ Getting started
+## Getting started
 
-🚧 WIP 🚧
+Periodically, issues within the docs repository will be labeled as 'Good First Issues'. These issues are ideally the first place to get started helping with the Charmed HPC documentation. If there are currently no 'Good First Issues' or none that fit your interest, feel free to reach out on Matrix and ask if the team has any suggestions. Otherwise, small PRs for grammar and formatting are always a good place to start.
```

Comment:
Just a couple suggested revisions here. Big thing is changing it from 'Good First Issues' to `good first issue` since that how our new issue labels are styled:

```suggestion
Periodically, issues within the docs repository will be labeled as `good first issue`. These issues are ideally the first place to get started helping with Charmed HPC's documentation. If there are currently no issues labeled `good first issue`, or none of the issues fit your interest, feel free to reach out on [Matrix](https://matrix.to/#/#hpc:ubuntu.com) and ask if the team has any suggestions. Otherwise, small PRs for grammar and formatting are always a good place to start.
```